src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -1581,252 +1581,381 @@ bool AletheProofPostprocessCallback::update(Node res,
+              d_resPivots ? std::vector<Node>{compSimp, d_true, geq, d_false}
+                          : std::vector<Node>(),
+              *cdp);
+          // Now we build the final resultion
+          success &= addAletheStep(
+              AletheRule::RESOLUTION,
+              res,
+              nm->mkNode(Kind::SEXPR, d_cl, res),
+              {leq, laDiseqCl, resPi1Conc},
+              d_resPivots ? std::vector<Node>{leq, d_true, leqInverted, d_false}
+                          : std::vector<Node>(),
+              *cdp);
+          break;
         }
-        else
+        case Kind::GT:
         {
-          equal = children[1];
-          lesser = children[0];
+          Trace("alethe-proof") << "..case GT\n";
+          Node geq, notEq;
+          Kind kc0 = children[0].getKind();
+          if (kc0 == Kind::GEQ
+              || (kc0 == Kind::NOT && children[0][0].getKind() == Kind::LT))
+          {
+            geq = children[0];
+            notEq = children[1];
+          }
+          else
+          {
+            geq = children[1];
+            notEq = children[0];
+          }
+          Assert(notEq.getKind() == Kind::NOT
+                 && notEq[0].getKind() == Kind::EQUAL);
+          // it may be that the premise supposed to be (>= x c) is actually
+          // (not
+          // (< x c)). In this case we use that premise to deriv (>= x c), so
+          // that the reconstruction below remains the same
+          if (geq.getKind() != Kind::GEQ)
+          {
+            Assert(geq.getKind() == Kind::NOT && geq[0].getKind() == Kind::LT);
+            Node notLt = geq;
+            geq = nm->mkNode(Kind::GEQ, x, c);
+            //  @pa: (= (< x c) (not (<= c x)))
+            //  @pb: (< x c)
+            //  @pc: (<= c x)
+            //  notLT : (not @pb)
+            //
+            // PI_a:
+            //
+            // ----- comp_simp  --------------------- equiv_pos1     ---- notLT
+            //  @pa             (cl (not @pa) @pb (not (not @pc)))   (not @pb)
+            // ------------------------------------------------------ resolution
+            //              (cl (not (not @pc)))
+            //
+            //
+            // PI_b:
+            //
+            //  ------------------------------ NOT_NOT -------------------- PI_a
+            //  (cl (not (not (not @pc))) @pc)         (cl (not (not @pc)))
+            // ------------------------------------------------------ resolution
+            //                             @pc
+            //
+            // PI_c:
+            //
+            //  @pd: (= (>= x c) (<= c x))
+            //
+            // ----- comp_simp  -------------------------- equiv_pos1  --- PI_b
+            //  @pd             (cl (not @pd) (>= x c) (not @pc))      @pc
+            // ------------------------------------------------------ resolution
+            //              (cl (>= x c))
+            success &= addAletheStep(AletheRule::HOLE,
```

Comment:
Why is it a hole when you seem to have explained in the comment above this how to write the proof?

Reply:
I forgot I had not updated this. I'll fix.